cmake/CMakeLists.txt

```diff
@@ -613,6 +613,7 @@ if (onnxruntime_USE_TENSORRT)
   if (WIN32)
     set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /DELAYLOAD:nvinfer.dll")
   endif()
+  set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-deprecated-declarations")
```

Comment:
is this needed? can we fix the root cause? are we using deprecated functions in TRT EP?

Reply:
agree. I removed this line to make Windows build work. it's one of the changes I did for Windows build.

---
In reply to: [330582329](https://github.com/microsoft/onnxruntime/pull/1966#discussion_r330582329) [](ancestors = 330582329)